test/unittest/rtps/builtin/BuiltinDataSerializationTests.cpp

```diff
@@ -15,11 +15,12 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+#include <fastdds/core/policy/ParameterSerializer.hpp>
```

Comment:
Attention: this header is private. It was in the correct section but not alphabetically sorted.